packages/shared/src/components/modals/CommentBox.tsx

```diff
@@ -79,7 +87,8 @@ function CommentBox({
     }
 
     element.setAttribute('data-min-height', element.offsetHeight.toString());
-    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
+    fixHeight(element);
+    // needs to happen on load
```

Comment:
What does the above `element.setSelectionRange` actually do? I did not see any difference when removing practically the whole part:
```
if (value) {
    // eslint-disable-next-line no-param-reassign
    element.value = value;
    element.setSelectionRange(element.value.length, element.value.length);
}
```

Reply:
Just looking for ways to split up and simplify amount of work we need to do in this `useEffect`.

Reply:
> What does the above element.setSelectionRange actually do?

I think it is to focus on the very last part of the content.

Reply:
> Just looking for ways to split up and simplify amount of work we need to do in this useEffect.

This is going to be deleted/refactored on v6/v7 anyway, but I'm open to suggestions 🚀

Reply:
We don't have to delve on it too much just because it will be updated in v6/v7. My recommendation for this kind of cases would be to split up `useEffect` in multiple smaller effects that do specific things and then also have less or no dependencies in dependency array.

> It is to focus on the very last part of the content on load. QoL kind of thing.

For me it was focusing without that code also, so probably worth checking if actually needed down the line.

Reply:
Oh, I see. We've had some instances in the past on some devices where it was not working. But happy to take a look at it again once v6/v7 comes.

With regards to smaller `useEffects`, the reason we didn't break it up into smaller pieces, is we had a technical session with tkDodo in the past where he doesn't advise having multiple `useEffects` when it can be clustered into just one due to its performance issues. cc @rebelchris

Reply:
I would be interested to see what specific performance issues would be added here, without context on that session I would probably say it had something to do with state setters from useEffect since those of course introduce additional re-renders hence some performance impact. 

We can leave it like this and tackle down the line 🤝